Automattic/wp-calypso
# Remove data layer for Directly, do the requests directly

Removes the data layer that was wrapping the `Directly.initialize` and `Directly.askQuestion` functions and converts the action to thunks instead. That will allow us to remove the dependency on data layer code (middleware etc) from the Inline Help widget, making it smaller.

**How to test:**
Open the inline help popup, go to a free site that is supported only in public forums, and ask a question there. In development mode, don't worry about posting the question to a real site: there is a sandbox configured with the `config( 'directly_rtm_widget_environment' )` variable. Verify that it all works.
